src/libcollections/vec_deque.rs

```diff
@@ -32,12 +32,16 @@ use core::cmp;
 
 use alloc::heap;
 
+#[deprecated(since = "1.0.0", reason = "renamed to VecDeque")]
+#[unstable(feature = "collections")]
+pub use VecDeque as RingBuf;
+
 static INITIAL_CAPACITY: usize = 7; // 2^3 - 1
 static MINIMUM_CAPACITY: usize = 1; // 2 - 1
 
-/// `RingBuf` is a circular buffer, which can be used as a double-ended queue efficiently.
+/// `VecDeque` is a circular buffer, which can be used as a double-ended queue efficiently.
```

Comment:
Could you add the word "ring buffer" to the documentation so you can find this collection if you want something akin to a ring buffer?